se_exception_msg(message=""):
        raise NameError(message)

